nuget/lib/dependabot/nuget/update_checker/nuspec_fetcher.rb

```diff
@@ -26,17 +26,9 @@ def self.fetch_nuspec_from_repository(repository_details, package_id, package_ve
 
         nuspec_xml = nil
 
-        if azure_package_feed?(feed_url)
-          # this is an azure devops url we can extract the nuspec from the nupkg
-          package_data = NupkgFetcher.fetch_nupkg_buffer_from_repository(repository_details, package_id,
-                                                                         package_version)
-          return if package_data.nil?
-
-          nuspec_string = extract_nuspec(package_data, package_id)
-          nuspec_xml = Nokogiri::XML(nuspec_string)
-        else
+        if feed_supports_nuspec_download?(feed_url)
           # we can use the normal nuget apis to get the nuspec and list out the dependencies
-          base_url = feed_url.gsub("/index.json", "-flatcontainer")
+          base_url = repository_details[:base_url].delete_suffix("/")
```

Comment:
This change is what allows us to pull `.nuspec` files directly from Azure DevOps.